phpBB/language/en/acp/permissions_phpbb.php

```diff
@@ -107,6 +107,7 @@
 // Forum Permissions
 $lang = array_merge($lang, array(
 	'ACL_F_LIST'		=> 'Can see forum',
+	'ACL_F_LIST_TOPICS' => 'Can see topics',
```

Comment:
I would find this really confusing.

- Can see forum
- Can see topics
- Can read forum

But you can read the forum if you can see the topics, right? Right? #AdvocatusDiaboli

Reply:
Not really, no. If you can see topics but can't read the forum then you won't be able to open individual topics even though you'd see the titles.